ppb/events.py

```diff
@@ -98,6 +109,23 @@ class ButtonReleased:
     scene: Scene = None
 
 
+@dataclass
+class ChangeScene:
```

Comment:
I feel like this is a insufficiently distinct name from `ReplaceScene`. Either make it more clear this is adding to the stack or explicitly document it?

Reply:
First one I named. We can just make it `StartScene`?